src/operations/property-filter.ts

```diff
@@ -85,7 +86,7 @@ export function propertyFilter<T>(
         key,
         operators,
         defaultOperator,
-      }: NonNullable<UseCollectionOptions<T>['propertyFiltering']>['filteringProperties'][0]
+      }: NonNullable<UseCollectionOptions<T, PropertyFilterProperty>['propertyFiltering']>['filteringProperties'][0]
```

Comment:
Can it be just `PropertyFilterProperty` here? It does not seem like we are using any other keys of this object

Reply:
Yes, will update 👍